### ISSUE TITLE
Q_5 never coloured

### ISSUE THREAD

**Issue by SoneaTheBest** [zjosua/anki-mc#93]
#### Describe the bug
Q_5 is never colored in when I look at the answer on the back of a card. 

#### To Reproduce
1. Create a card with 5 answer possibilities
2. Upon revealing the answer, Q_1 to Q_4 get the correct background color according to code, only Q_5 doesn't.

#### Expected behavior
Fifth answer should have the background color according to code.

#### What have you tried?
Since I changed the color meaning in my profile, I figured I must have messed up the programming somewhere, but the same mistake also appears in another profile where I haven't adjusted the code. Deleting and reinstalling the Add-on also didn't help. I looked at the code to see if there was an obvious Q_5 missing somewhere, but I know absolutely nothing about programing and was way out of my depth there. 

#### Screenshots
![grafik](https://user-images.githubusercontent.com/64635425/202803740-51556ddb-62f3-4560-bc30-9b113fb477e7.png)
![grafik](https://user-images.githubusercontent.com/64635425/202803766-09a0685d-4bae-48fd-8fd0-9400c0e8a33e.png)
![grafik](https://user-images.githubusercontent.com/64635425/202803794-94b3d277-7be2-439e-b3af-30d193c564e5.png)

#### Information about your set-up

On which platform(s) does the error occur?
- [x] Ankiweb
- [x] Desktop: Windows 10, 19044.2251
- [ ] Mobile: [don't use it]

#### Additional context
In the third screenshot it appears to also not correctly interpret the card if Q_5 was rightly chosen? I didn't notice that before, since I always put the right answer in Q_1. 

(Thanks for the wonderful add-on, even with this little bug, its working great and its an incredibly helpful addition to my Anki set-up!)

**Comment by 3ter** [comment]
Hey @SoneaTheBest , thank you for your report.

I strongly think it is based on a misconception though: Different profiles in Anki share the same addons! (And this addon replaces the note type only on installation and update)

If you want to test the note type isolated from your changes use `anki -b path/to/your/folder/of/choice` as discussed e.g. here: https://forums.ankiweb.net/t/seperating-anki-and-anki-source-code-build/21263/8.

**Comment by 3ter** [comment]
I just tested it again with a new folder and it works as expected so I'm going to remove the bug label.
Did you see that we've recently added an easier way of changing the coloring via the addon config (as described on the main page of this repo)?
![image](https://user-images.githubusercontent.com/11499929/202898858-da4471f2-0cd5-43e0-8676-4fde29622fc0.png)
![image](https://user-images.githubusercontent.com/11499929/202898861-17805f0d-ad5d-4a2b-a68f-334e4826f1d4.png)

**Comment by SoneaTheBest** [comment]
Thank you the answer!
I did a bit more playing around (e.g. testing cards to see what would happen). With some new cards I made it worked immediately. I looked at the cards I had screenshotted for you: Stil not working, i compared the input for both cards, they seemed to be the same. After retyping the answer code (the 0 1 0 0 0), its now magically working, although I didn't change it. I have no clue what the problem was, but it is working now. 
Thanks again!
